packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -218,7 +218,7 @@ class BuildWebCommand extends BuildSubCommand {
       );
     }
     if (!project.web.existsSync()) {
-      throwToolExit('Missing index.html.');
+      throwToolExit('Run flutter create . --platforms web');
```

Comment:
IMO only having this printed feels like it doesn't give the context as to why it is there.
The old message at least said that a file was missing.

What if we change it to something like:
```
This project is not configured for the web.

To configure this project for the web, run flutter create . --platforms web
```

Reply:
Yes I think that would be better since it give more context. I will go ahead and push the changes

Reply:
<img width="1141" alt="Screenshot 2025-03-16 at 1 25 25 PM" src="https://github.com/user-attachments/assets/2616796a-2a0b-48d6-b59d-ac99cbaa672b" />